docs/user_docs/spec_file_format.md

```diff
@@ -0,0 +1,35 @@
+# Spec file format
+**Will be rewritten** 
+This product uses CSV for file reading, this means that group definition is separated by a  ",". The spec for this product will have the following group definitions: name, ring quadrant, isNew, moved and description. For the CSV, this will look like:
+```
+name,ring,quadrant,isNew,moved,description
+```
+The reader will then translate these to structs which get translated to types using unmarshalling.
+
+The types foreach group:  
+```  
+- name        : string
+- ring        : string
+- quadrant    : string
+- isNew       : bool
+- moved       : int
+- description : string
+```
+
+Some of the groups is defined to take specific inputs. Groups can have these inputs:
+```
+- name        : The name of a data management, datastore, infrastructure or langauge.
+- ring        : The name of the ring it's place like adopt, trial, assess or hold.
+- quadrant    : The name of the quadrant like data management, datastore, infrastructure or langauge.
+- isNew       : True or false if the tech is new or not for the tech radar.
+- moved       : 0-3 depending on what ring the tech moved to. so going from hold to adopt is going from 0 to 3 is moved=3.
+- description : A small description on the tech.
```

Comment:
When "small" is used, it feels vague. How small is small? Is there a char limit? Do we want one? ;-;

Reply:
maybe change to "short" and maybe we do want to specify a char limit. Think something around 120 chars would be plenty maybe 250 if we wanna leave enough room for bigger tool descriptions

Reply:
If we can and want to set a char limit...

May I propose in the next sprint, that we experiment with describing some techstack from a previous project, then count, compare and discuss each our description's sizes?

We are 6 devs. If we pair up, we can describe 3 different types of "software tools", eg. a programming language, a database manager, and a data store.

I can imagine this might take around 15 minutes. Unless we get heated in the discussions :')

Reply:
I mean guess it would also be worth discussing whether a char limit would be too limiting for the users since there may be cases where a description just has to be slightly longer. But would say lets do as you suggested and talk about it as a group